web/client/selectors/__tests__/localConfig-test.js

```diff
@@ -43,11 +42,15 @@ describe('localConfig selectors', () => {
 
     it('pluginSelectorCreator for dashboard', ()=>{
         const loadedConfig = (pluginsSelectorCreator('dashboard')(stateMocker(localConfigLoaded(LOCAL_CONFIG))));
-        expect(includes(loadedConfig, 'DashboardSave')).toBe(true);
-        expect(includes(loadedConfig, 'DashboardSaveAs')).toBe(true);
-        expect(filter(loadedConfig, { "name": "Share"})[0]).toContain({ "name": "Share"});
-        expect(filter(loadedConfig, { "name": "Share"})[0].cfg).toContain({ "embedPanel": false});
-        expect(filter(loadedConfig, { "name": "Share"})[0].cfg).toContain({ "advancedSettings": false});
+        expect(find(loadedConfig, { "name": 'DashboardSave'})).toContain({ "name": 'DashboardSave'});
```

Comment:
Instead of testing the localConfig.json that will be restructured soon test the plugin (if not too hard).